rules/windows/execution_suspicious_dotnet_compiler_parent_process.toml

```
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/08/21"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Suspicious dotNet Code execution. Verify process details such as commandline, parent process and network
+connections.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious dotNet code execution"
```

```suggestion
name = "Suspicious .Net code execution"
```

May need to expand the name here since we may have more than 1 suspicious .net rule